Concept_Templates/single_crystal.md

```diff
@@ -0,0 +1,36 @@
+## Knowledge domain resources
+
+### Related
+
+https://dictionary.iucr.org/crystal: historical, reciprocal and direct space definitions. 
+Expert discussions also raised the issue of different dimensionality crystals, quasicrystals and so on.
+
+https://en.wikipedia.org/wiki/Crystal: information as above, as well as history of the term,
+some discussion of various types of crystal including quasicrystals and crystal defects.
+
+### Comments
+
+The vast majority of crystal structures are formed of arrangements of atoms that are
+periodic in 3 dimensions (400,000 according to Wikipedia reference 20). Almost all
+measurements on samples are modelled assuming the sample is a single crystal.
+
+## General Concept Info
+
+### IRI: TBA
+### OWL type: TBA
+### Concept elucidation
+A sample of a solid substance that has been well-described by using a crystalline model
```

Comment:
Crystalline model should be defined as an own concept.

Reply:
Yes indeed, see #12 .